src/modules/airdrop/services/airdrop-service.ts

```diff
@@ -60,9 +61,15 @@ export class AirdropService {
     let welcomeTravellerCompleted = false;
 
     if (welcomeTravellerEligible) {
-      const depositCount = await this.depositRepository.count({
-        where: { depositorAddr: checksumAddress, status: "filled" },
-      });
+      const depositCountQuery = await this.depositRepository
+        .createQueryBuilder("deposit")
+        .innerJoin(Token, "token", "deposit.tokenId = token.id")
+        .where("deposit.depositorAddr = :depositorAddr", { depositorAddr: checksumAddress })
+        .andWhere(
+          "(token.symbol = 'USDC' and deposit.amount / token.decimals >= :usdAmount) or (token.symbol = 'WETH' and deposit.amount / token.decimals >= :wethAmount)",
+          { usdAmount: 150, wethAmount: 0.1 },
+        );
+      const depositCount = await depositCountQuery.getCount();
```

Comment:
check if the depositor address has a USDC deposit >= 150 or a WETH deposit >= 0.1